parquet/shield-study-addon.parquetmr.txt

```diff
@@ -0,0 +1,73 @@
+message shield-study-addon {
+    required group application {
+        required binary architecture (UTF8);
+        required binary buildId (UTF8);
+        required binary channel (UTF8);
+        required binary name (UTF8);
+        required binary platformVersion (UTF8);
+        required binary version (UTF8);
+    }
+    required binary clientId (UTF8);
+    required binary creationDate (UTF8);
+    required group environment {
```

Comment:
It came derived from the json schema.  I imagine it's that we expect env to always exist (because adding a key is safe), but we aren't sure it can actually populate the stuff (because the code for generating the stuff can fail for weird reasons).